src-docs/src/views/code/code_example.js

```diff
@@ -39,6 +41,23 @@ export const CodeExample = {
   title: 'Code',
   intro: (
     <>
+      <EuiCallOut title="EUI provides support for static code">
+        <p>
+          <strong>EuiCode</strong> and <strong>EuiCodeBlock</strong> are
+          intended to render static lines or blocks of code in read-only
+          contexts.
+          <br />
+          If you need capabilities to edit, or want to print long code (e.g.,
+          printing JSON from an API), use{' '}
+          <EuiLink
+            external
+            href="https://github.com/elastic/kibana/tree/master/src/plugins/kibana_react/public/code_editor">
+            <strong>CodeEditor</strong> (within Kibana)
+          </EuiLink>
+          .
```

Comment:
```suggestion
          If you need capabilities to edit, or want to print long code (e.g.,
          printing JSON from an API), we recommend installing a version of Monaco. If you are building
          within the Kibana platform, you can use their{' '}
          <EuiLink
            href="https://github.com/elastic/kibana/tree/master/src/plugins/kibana_react/public/code_editor">
            <strong>CodeEditor</strong>
          </EuiLink>.
          .
```

Or something like that. Basically, not all consumers are building Kibana and we should point them to our recommended approach. YOu can also link to Monaco as well.